dashboard/app/views/home/_homepage.html.haml

```diff
@@ -12,7 +12,7 @@
 - if current_user
   - homepage_data[:providers] = current_user.providers
   - if current_user.teacher?
-    - homepage_data[:hocLaunch] = DCDO.get('hoc_launch', CDO.default_hoc_launch)
+    - homepage_data[:hocLaunch] = DCDO.get('hoc_mode', CDO.default_hoc_mode) == 'actual-hoc' ? 'actual-hoc' : DCDO.get('hoc_launch', CDO.default_hoc_launch)
```

Comment:
I don't understand this line. If we're in hoc_mode `actual-hoc` then we're in hoc_launch `actual-hoc`? Shouldn't it only return `mc` or `dance`?

Reply:
The teacher announcement has three variations, so I just set one of those three here: "actual-hoc", "mc", or "dance".  Sorry the naming's a bit confusing; I just didn't want to plumb through another parameter for this one case.

Reply:
Oh, I think I see what you're doing. Setting `hoc_launch` to `dance` is for the Wednesday flip and then setting `hoc_launch` to `actual-hoc` is for the Saturday flip?

Reply:
We will set `hoc_launch` to `dance` on Wednesday, but we'll leave it intact and just set `hoc_mode` to `actual-hoc` on Saturday.